AFQ/viz_libs/utils.py

```diff
@@ -0,0 +1,186 @@
+from collections import OrderedDict
```

Comment:
Would be good to have some docstrings in this module, even for the helper functions.